public/js/action-list.js

```diff
@@ -163,6 +191,22 @@
                 ).addClass('active');
             }
         }
+
+        var $activeItems = $list.find('.list-item.active');
+
+        if ($activeItems.length) {
+            var container = $list.closest('.container');
+
+            if (! container.children('.footer').length) {
+                container.append('<div class="footer" data-action-list-automatically-added></div>');
+            }
+
+            var footer = container.children('.footer');
+            var label = $list.data('icinga-multiselect-count-label').replace('%d', $activeItems.length);
+
+            footer.prepend('<div class="selection-count"></div>');
+            footer.children('.selection-count').append('<span class="selected-items">' + label + '</span>');
```

Comment:
Please combine both lines. You can include the span with the `.selection-count` div already.